src_utils/deepspee.py

```diff
@@ -0,0 +1,403 @@
+# Copyright 2020 The HuggingFace Team. All rights reserved.

```

Comment:
renamed this file bc modeling_utils.py was having trouble importing from this file. Instead, it kept importing from the deepspeed library